common/version.go

```diff
@@ -0,0 +1,7 @@
+package common
+
+// Set by ldflags
+var GitCommit, GitBranch, GitState, GitSummary, BuildDate string
```

Comment:
In go-textile, these would be populated by github.com/ahmetb/govvv, which we probably don't need/want here. I'm not sure the best way to do a version file these days... maybe @jsign has some opinions.

Reply:
Not really strong ideas here. Didn't know about `govvv` but seems a helper for passing `-ldflags` to the linker.
I think the only improvement that comes to my mind is automating that `Version` variable with something like: `git tag | sort -V | tail -n 1`, to get the latest tagged version.

Reply:
okay, pushed up a change that drops common/version and just pulls from latest git tag!